tests/core/test_playback.py

```diff
@@ -214,7 +211,33 @@ def test_get_current_track_next(self):
         self.replay_events()
 
         current_track = self.core.playback.get_current_track()
-        self.assertEqual(current_track, self.tracks[1])
+        assert current_track == self.tracks[1]
+
+    @pytest.mark.parametrize('repeat,random,single,consume,index,result', [
+        (False, False, False, False, 0, 1),
+        (False, False, False, False, 2, None),
+        (True, False, False, False, 0, 1),
+        (True, False, False, False, 2, 0),
+    ])
+    def test_next_all_modes(
+            self, repeat, random, single, consume, index, result):
+        tl_tracks = self.core.tracklist.get_tl_tracks()
+
+        self.core.playback.play(tl_tracks[index])
+        self.replay_events()
+        self.core.tracklist.repeat = repeat
+        self.core.tracklist.random = random
+        self.core.tracklist.single = single
+        self.core.tracklist.consume = consume
```

Comment:
These properties are deprecated. Use `set_repeat(repeat)` and friends.

I have an upcoming PR that fixes this for the rest of the file, so just fix it for these new tests.